server/methods/registerUser.coffee

```diff
@@ -29,6 +29,13 @@ Meteor.methods
 
 		try
 			if userData.email
+				if RocketChat.settings.get('Verification_Customized')
+					subject = RocketChat.placeholders.replace(RocketChat.settings.get('Verification_Email_Subject') || '')
+					html = RocketChat.placeholders.replace(RocketChat.settings.get('Verification_Email') || '')
+					Accounts.emailTemplates.verifyEmail.subject = (userModel) ->
+						return subject
+					Accounts.emailTemplates.verifyEmail.html = (userModel, url) ->
+						return html.replace('[Verification_Url]', url);
```

Comment:
All of the `.replace(String, 'something')` will only replace it once. This needs to be converted into regex for it to replace every instance of that string. So, in this case `html.replace(/\[Verification_Url\]/g, url)`. Just thinking that someone could put that in a url and then below put it again if a client can only read the text and not the anchors.